sympy/solvers/solveset.py

```diff
@@ -739,7 +741,7 @@ def solveset(f, symbol=None, domain=S.Complexes):
         A set of values for `symbol` for which `f` is True or is equal to
         zero. An `EmptySet` is returned if `f` is False or nonzero.
         A `ConditionSet` is returned as unsolved object if algorithms
-        to evaluatee complete solution are not yet implemented.
+        to evaluate complete solution are not yet implemented.
```

Comment:
Fixing spelling mistakes is fine. It's usually better to do it in a separate commit, but if it's near the PR change you don't need a separate PR.